demo/demo-spring-boot-transport/demo-spring-boot-pojo-server/src/main/java/io/servicecomb/demo/pojo/server/TestImpl.java

```diff
@@ -19,24 +19,39 @@
 import java.util.Arrays;
 import java.util.List;
 
+import javax.ws.rs.core.MediaType;
+
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+
 import io.servicecomb.demo.server.Test;
 import io.servicecomb.demo.server.TestRequest;
 import io.servicecomb.demo.server.User;
 import io.servicecomb.provider.pojo.RpcSchema;
 import io.servicecomb.swagger.invocation.exception.InvocationException;
 
 @RpcSchema(schemaId = "server")
+@RequestMapping(path = "/pojo/rest", produces = MediaType.APPLICATION_JSON)
```

Comment:
why RpcSchema need this annotations?

Reply:
I keep this annotation in code for the purpose that users can transform this demo back into a Spring Boot demo more conveniently.(And they can compare this two kind of framework)
But actually this annotation is redundant, I will remove it and create another PR.